src/windowsdesktop/src/sfx/Microsoft.WindowsDesktop.App.Runtime.sfxproj

```diff
@@ -41,24 +35,34 @@
     <FrameworkListFileClass Include="System.Windows.Forms.Design.resources.dll" Profile="WindowsForms" />
     <FrameworkListFileClass Include="System.Windows.Forms.Primitives.resources.dll" Profile="WindowsForms" />
     <FrameworkListFileClass Include="System.Windows.Forms.resources.dll" Profile="WindowsForms" />
-    <FrameworkListFileClass Include="System.Windows.Input.Manipulations.resources.dll" Profile="WindowsForms" />
     <FrameworkListFileClass Include="System.Private.Windows.Core.dll" Profile="WindowsForms" />
```

Comment:
As I noted in https://github.com/dotnet/windowsdesktop/pull/4227#pullrequestreview-2353112435 this is the wrong way of doing this, Windows Forms owns the list, and it should be coming from dotnet/winforms instead of being manually maintained here.

Reply:
Yes, this is what https://github.com/dotnet/windowsdesktop/issues/4796 describes. Currently https://github.com/dotnet/winforms/blob/main/pkg/Microsoft.Private.Winforms/sdk/dotnet-windowsdesktop/System.Windows.Forms.FileClassification.props is only for the ref pack, we'll need to make adjustments to have it also be used for the runtime pack here.